oap-logstream/src/main/java/oap/logstream/disk/AbstractFinisher.java

```diff
@@ -49,7 +49,7 @@ public void run() {
         var bucketStartTime = timestamp.currentBucketStartMillis();
         var elapsed = DateTimeUtils.currentTimeMillis() - bucketStartTime;
         if( elapsed < safeInterval )
-            log.debug( "not safe to process yet ({}ms), some of the files could still be open, waiting...", elapsed );
+            log.debug( "not safe to process yet ({} ms), some of the files could still be open, waiting...", elapsed );
```

Comment:
SI brochure recommends the space between numbers in units however this space by English rules should not be breakable. Since in practice it is impossible to enforce the usual practice is not to put the space in this situation.